src/domain/transfer/models/transfers-read-model.js

```diff
@@ -1,16 +1,35 @@
+/*****
```

Comment:
Please, rename filename and all references to transfer-read-model.js
Currently the test filename does not match (transfer-read-model.test.js)